exchanges/stream/websocket.go

```diff
@@ -677,76 +680,70 @@ func (w *Websocket) CanUseAuthenticatedWebsocketForWrapper() bool {
 
 // SetWebsocketURL sets websocket URL and can refresh underlying connections
 func (w *Websocket) SetWebsocketURL(url string, auth, reconnect bool) error {
```

Comment:
Back to front-review-comments 🤦 
See my comment below on GetWebsocketURL.
I actually vote for 2 methods on this too, even though it changes the API.

Trying to think about how this pattern would look like for a router, but I'm about a week away from putting that up on the drawing board. Still mulling it over.